core-web/libs/sdk/client/src/lib/utils/page/common-utils.ts

```diff
@@ -42,9 +41,7 @@ export const getPageRequestParams = ({
     const finalParams: Record<string, unknown> = {};
     const dotMarketingPersonaId = copiedParams['com.dotmarketing.persona.id'] || '';
 
-    if (copiedParams['mode']) {
-        finalParams['mode'] = copiedParams['mode'];
-    }
+    finalParams['mode'] = copiedParams['mode'] || 'LIVE';
```

Comment:
Always live? I think this is about the fix we discussed on the endpoint right?

Reply:
Yup, is to always list as live from the client